lib/qrolo.dart

```diff
@@ -27,526 +28,277 @@ import 'package:qrolo/src/html/media/utilities/is_media_device_camera_available.
 import 'package:qrolo/src/jsqr.dart' show jsQR;
 
 import 'package:js/js.dart' show JS, anonymous;
+import 'package:qrolo/src/media.dart';
```

Comment:
Slight difference in project scaffold,
Otherwise remainder should be considered a bulk file replacement to a working state rather than line differences